core/src/test/java/org/apache/calcite/test/RelMetadataTest.java

```diff
@@ -1302,16 +1461,15 @@ private static ImmutableBitSet bitSetOf(int... bits) {
     sql("select a1.empno, a2.empno\n"
         + "from emp a1 join emp a2 on (a1.empno=a2.empno)")
         .convertingProjectAsCalc()
-        .assertThatUniqueKeysAre(bitSetOf(0), bitSetOf(1), bitSetOf(0, 1));
+        .assertThatUniqueKeysAre(bitSetOf(0), bitSetOf(1));
   }
 
   @Test void calcMultipleColumnsAreUniqueCalc3() {
     sql("select a1.empno, a2.empno, a2.empno\n"
         + " from emp a1 join emp a2\n"
         + " on (a1.empno=a2.empno)")
         .convertingProjectAsCalc()
-        .assertThatUniqueKeysAre(bitSetOf(0), bitSetOf(0, 1), bitSetOf(0, 1, 2),
-            bitSetOf(0, 2), bitSetOf(1), bitSetOf(1, 2), bitSetOf(2));
+        .assertThatUniqueKeysAre(bitSetOf(0), bitSetOf(1), bitSetOf(1, 2), bitSetOf(2));
```

Comment:
Ideally this would be:
```
        .assertThatUniqueKeysAre(bitSetOf(0), bitSetOf(1), bitSetOf(2));
```
I thought digging into this was scope creep.